sailor/utils/config.py

```diff
@@ -14,7 +14,7 @@
 LOG = logging.getLogger(__name__)
 LOG.addHandler(logging.NullHandler())
 
-CONFIG_PROPERTIES = ('asset_central', 'sap_iot')
+CONFIG_PROPERTIES = ('asset_central', 'sap_iot', 'pai')
```

Comment:
We need to make it optional. 

Note to self: it's possible with the namedtuple object to supply default values to the rightmost arguments. Sounds wonky, but it works.

Reply:
this seemed to work.. 
class SailorConfig(namedtuple('SailorConfig', CONFIG_PROPERTIES, defaults=(None,) * len(CONFIG_PROPERTIES))):
#class SailorConfig(namedtuple('SailorConfig', CONFIG_PROPERTIES)):